app/helpers/results_helper.rb

```diff
@@ -36,16 +36,8 @@ def pdf_friendly_h2(text, size, is_pdf, additional_class = nil)
     end
   end
 
-  def pdf_friendly_h3(text, is_pdf)
```

Comment:
@MazOneTwoOne Could you have a look at this. I was getting failing code coverage for if part of this method. From what I can see we use it only in app/views/estimates/_controlled_work_next_steps.html.slim

But I cannot work out what combination of attributes a case needs and what action has to be undertaken to generate a PDF that uses this styling. Hence I have removed it to see if anything breaks and it doesnt appear to have had any impact. I would welcome your thoughts on it though.

Reply:
@willc-work this gets used when generating the PDF of the results page for a controlled work check. The reason you got a code coverage error is that you've changed the `downloads_spec` tests so that now you're not generating a download for a controlled work check.

And the reason you've ended up doing that is it looks like you've accidentally removed the 'Download' option on the results screen for eligible controlled checks.

Reply:
thanks @patrick-laa